test/integration/ggrc/models/test_program.py

```diff
@@ -27,3 +27,25 @@ def test_put_empty_audits(self):
     self.assert200(response)
     audit = self.refresh_object(all_models.Audit, id_=self.audit_id)
     self.assertIsNotNone(audit)
+
+  def test_delet_with_audits(self):
```

Comment:
please fix typo "delet**e**"

Reply:
@belovmd  for these kind of changes, github has a ` ```suggestion ` box that you can use like this:
```suggestion
  def test_delete_with_audits(self):
```